README.md

```diff
@@ -1,20 +1,122 @@
+
+Currently, we support datasets having numerical and categorical features. The following table provides the scenarios supported for each of the four responsible AI insights:-
+
+| RAI insight | Binary classification | Multi-class classification | Multilabel classification | Regression | Timeseries forecasting | Categorical features | Text features | Image Features | Recommender Systems | Reinforcement Learning |
+| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | -- |
+| Explainability | Yes | Yes | No | Yes | No | Yes | No | No | No | No |
+| Error Analysis | Yes | Yes | No | Yes | No | Yes | No | No | No | No |
+| Causal Analysis | Yes | No | No | Yes | No | Yes (max 5 features due to expensiveness) | No | No | No | No |
+| Counterfactual | Yes | Yes | No | Yes | No | Yes | No | No | No | No |
+
+This is all available via Python SDK or CLI.
+
+## Set Up
+In this section, we will go over the basic setup steps that you need in order to generate Responsible AI insights for your models from SDK v2, CLI v2 and visualize the generated Responsible AI insights in [AML studio](https://ml.azure.com/).
+
+### Create an Azure subscription
+Create an Azure workspace by using the [configuration notebook](https://github.com/Azure/MachineLearningNotebooks/blob/master/configuration.ipynb)
+
+### Install the required packages
+In order to install `azureml-responsibleai` package you will need a python virtual environment. You can create a python virtual environment using `conda`.
+```c
+conda create -n azureml_env python=3.8
+activate azureml_env
+```
+
+After activating your environment, if this is your first time running the RAI Dashboard in private preview then continue to the [setup instructions](https://github.com/Azure/RAI-vNext-Preview/blob/mergePrivatePreview_MAD/docs/Setup.md) to do a one-time setup for your workspace.
+
+
+
+### Generating Responsibleai AI Dashboard insights
+Once you have created an Azure workspace and registered your components in the one-time setup above, you can create a Responsible AI dashboard via the CLI or SDK. Start here for `examples` [folder](examples) to get started.
+
+### Viewing your Responsible AI Dashboard in the AzureML studio portal
+After generating the Responsible AI insights, you can view them in your associated workspace in AzureML studio, under your model registry.
+
+![01](images/01_model_registry.png)
+1. Go to your model registry in your AzureML studio workspace
+2. Click on the model you've uploaded your Responsible AI insights for
```

Comment:
Ending this sentence with a preposition feels odd rather than attention grabbing.

Reply:
nix "for" - maybe "to" >?

Reply:
I've switched the sentence around.